Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -1177,8 +1177,7 @@ struct LengthOrPercentRawConsumer {
     using FunctionToken = SameTokenMetaConsumer<
         Transformer,
         PercentRawKnownTokenTypeFunctionConsumer,
-        LengthRawKnownTokenTypeFunctionConsumer
-    >;
+        LengthRawKnownTokenTypeFunctionConsumer>;
```

Comment:
This doesn't seem like it is part of the change. Should leave off these unrelated style or whitespace changes.